From: Iain Lane <laney@debian.org>
Date: Thu, 26 May 2011 10:52:24 +0100
Subject: Relax build-dependency on time

The versions specified for the Build-Depend on time were too strict;
GHC's version works fine.
---
 convertible.cabal |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

--- a/convertible.cabal
+++ b/convertible.cabal
@@ -50,7 +50,7 @@

 library
   if flag(splitBase)
-    Build-Depends: base>=3 && <5, old-time, time>=1.1.2.4 && <=1.2.0.5,
+    Build-Depends: base>=3 && <5, old-time, time>=1.1.2.4 && <1.3.0.0,
      bytestring, containers, old-locale
     if flag(time_gte_113)
       Build-Depends: time>=1.1.3
